src/main.py

```diff
@@ -174,34 +176,39 @@ def open_pr(subject, body, branch):
     check_call(("git", "push", "-u", remote, branch))
 
     pr = origin_repo.create_pull(
-        subject,
-        pr_message,
-        base,
-        head,
-        maintainer_can_modify=True,
+        subject, pr_message, base, head, maintainer_can_modify=True,
     )
     log.info("Opened pull request %s", pr.html_url)
 
 
 def main():
-    types = ['all'] + list(ExternalData.TYPES)
+    types = ["all"] + list(ExternalData.TYPES)
     parser = argparse.ArgumentParser()
-    parser.add_argument("manifest", help="Flatpak manifest to check",
-                        type=os.path.abspath)
-    parser.add_argument('-v', '--verbose', help='Print debug messages',
-                        action="store_true")
-    parser.add_argument("--update",
-                        help="Update manifest(s) to refer to new versions of "
-                             "external data - also open PRs for changes unless "
-                             "--commit-only is specified",
-                        action="store_true")
-    parser.add_argument("--commit-only",
-                        help="Do not open PRs for updates, only commit changes "
-                             "to external data (implies --update)",
-                        action="store_true")
-    parser.add_argument('--filter-type',
-                        help='Only check external data of the given type',
-                        choices=types, default='all')
+    parser.add_argument(
+        "manifest", help="Flatpak manifest to check", type=os.path.abspath
+    )
+    parser.add_argument(
+        "-v", "--verbose", help="Print debug messages", action="store_true"
+    )
+    parser.add_argument(
+        "--update",
+        help="Update manifest(s) to refer to new versions of "
+        "external data - also open PRs for changes unless "
```

Comment:
I really, really loathe this aspect of `black`'s formatting, but still not enough to do anything about it like “file a bug”.

Reply:
Yeah, I'm not particularly happy about it either, but it hand-waves the entire class of code formatting issues like `gofmt` does. I can live with opinionated tool that makes me not think about it even if I dislike some things.